G1ANT.Addon.Images/G1ANT.Addon.Images.Tests/Commands/image.find/ShouldFindImageLocation_WhenSearchForImage.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿addon images version 2.205.17304.1000
+
+♥guid = ⊂Guid.NewGuid().ToString()⊃
+♥imageFile = ♥environment⟦temp⟧\♥guid.jpg
+file.download https://jeremykun.files.wordpress.com/2012/01/img49.png filename ♥imageFile
+program mspaint.exe arguments ♥imageFile
```

Comment:
mspaint.exe should be closed after test

Reply:
keyboard window ♥activeWindow text ⋘ALT+F4⋙